Documentation/SystemModules.md

```diff
@@ -17,7 +17,8 @@ Lets put this code in a directory called `example`:
 
     $ mkdir example
     $ cd example
-    example$ touch main.swift Package.swift
+    example$ swift package init --type exec
+    example$ touch main.swift
```

Comment:
`touch main.swift` is wrong here, the init command creates the source file (but I believe creates it in Sources, we should make sure these instructions work)